Module-7/HomeWork7/Homework7/Views/ModularContainerView.swift

```diff
@@ -0,0 +1,118 @@
+//
+//  ModularContainerView.swift
+//  Created by Mark Renaud (2023).
+//
+
+import SwiftUI
+
+struct ModularContainerView<StoredType: Codable, ProjectedPropertyType: BidirectionalCollection>: View {
+    @StateObject var store: ModularStore<StoredType, ProjectedPropertyType>
```

Comment:
Please make sure to always mark `@StateObject` property  as `private` too (just like with `@State`).
`@StateObject` properties should always be private to container where they are created and not be passed from the outside. This way SwiftUI knows how to handle them properly in terms of memory management. In other words, if this is something shared with other Views, mark it as `@ObservedObject` or `@EnvironmentObject`or if it's just local to this View, use passed values to initialize the `@StateObject` here.
You can find more info here: https://developer.apple.com/documentation/swiftui/stateobject

Reply:
Thanks for the catch.  Agreed.  Updated to follow guidelines.